introduction/welcome_to_java/Solution.java

```diff
@@ -1,5 +1,4 @@
 public class Solution {
-
     public static void main(String[] args) {
         /* Enter your code here. Print output to STDOUT. Your class should be named Solution. */
```

Comment:
This is a TODO, remove since it's done.
Also generally use // - style comments inside a method.